test/Microsoft.Health.Dicom.Tests.Integration/Persistence/SqlServerSchemaUpgradeTests.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace Microsoft.Health.Dicom.Tests.Integration.Persistence
 {
-    public class SqlServerSchemaUpgradeTests
+    public class SqlServerSchemaUpgradeTests1
```

Comment:
Tests within a collection (here created implicitly by a class) are always run sequentially - there's no built-in way to override this behavior. These tests take several minutes each on the build agent, so splitting them into separate classes is the easiest way to achieve parallelization. There is a workaround we could try, or just wait for xUnit 3: https://github.com/xunit/xunit/issues/1986